packages/block-library/src/block-keyboard-shortcuts/index.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * WordPress dependencies
+ */
+import { useEffect } from '@wordpress/element';
+import { useSelect, useDispatch } from '@wordpress/data';
+import {
+	useShortcut,
+	store as keyboardShortcutsStore,
+} from '@wordpress/keyboard-shortcuts';
+import { __ } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+import { store as blockEditorStore } from '@wordpress/block-editor';
+
+function BlockKeyboardShortcuts() {
```

Comment:
In this component registration and usage is merged. They were in different components before. The reason is that in some contexts, we want to register the shortcuts (show them in the keyboard shortcuts modal) but "disable" them in some contexts (view mode in the site editor for instance). 

I'm not sure whether this creates issues for this PR, it needs to be tested.

Reply:
It seems the `Block` prefix is redundant here as well, we're already in the block library package.

Reply:
I can rename it but then I'd have to import it 'as' a different name you know, because other components are already named KeyboardShortcuts.

Reply:
yeah, we do that already for other things. I don't care much though, it's a private API anyway.

Reply:
> In this component registration and usage is merged. They were in different components before. 

I think they were separated only in edit-site, where the `use` was [here](https://github.com/WordPress/gutenberg/blob/cca1f55f048f027df49f0c08e757e73f73a54193/packages/edit-site/src/components/keyboard-shortcuts/edit-mode.js#L71-L83)? Not sure that makes a difference, because they are within [a check for isEditMode](https://github.com/WordPress/gutenberg/blob/678938fc22d4f67c2890e73623adb7457125351b/packages/edit-site/src/components/editor/index.js#L283-L290)?

Reply:
yes, for this PR it probably doesn't change much.